src/components/cards/VideoCard.jsx

```diff
@@ -62,11 +77,11 @@ const VideoCard = ({details,setOpenModal, setNewVideo}) => {
           alt="profile"
         />
         <div className="coloumn-flex title-container">
-          <p className="video-main-title">
-            {title}
-          </p>
+          <p className="video-main-title">{title}</p>
           <p className="video-sub-title">{creator}</p>
-          <p className="video-sub-title">{statistics["viewCount"]}M views • {statistics["likeCount"]}K likes</p>
+          <p className="video-sub-title">
+            {statistics["viewCount"]}M views • {statistics["likeCount"]}K likes
```

Comment:
can destruct this `statistics` object above.

Reply:
ok